utils/storage/cookie.js

```diff
@@ -75,6 +75,20 @@ class CookieLocal {
       return false;
     }
   }
+
+  /**
+   * Function to check cookie support exists or not
+   * @returns boolean
+   */
+  IsCookieSupported() {
```

Comment:
Can you re-use this method for the above cookie check?

Reply:
done